lib/cube_browser/__init__.py

```diff
@@ -431,7 +431,7 @@ class Contourf(Plot2D):
     """
     def draw(self, cube):
         self.element = iplt.contourf(cube, axes=self.axes, coords=self.coords,
-                                     **self.kwargs)
+                                     extend='both', **self.kwargs)
```

Comment:
> Could you elaborate on the artifacts that are being removed by adding extend='both'?

I think there are contouring artifacts that shouldn't be there which can be seen, e.g.:
#128 
this only seems to affect contourf, not contour

However, that is not the primary benefit of this PR.  This ensures that once the levels are set, all data will be provided with a fill colour. 
This is important whether or not we have contour artifacts which need tidying up.  Otherwise, we are likely to get lots of white areas, particularly for the contourf, as sliders are interacted with